.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "airbnb",
+
+    "rules": {
+        "indent": [2, 2],
```

Comment:
That can be automated with tools like jscs that can re-indent the whole project, but i'd do that in a separate PR...